build/__snapshots__/features-meta.json

```diff
@@ -669,6 +669,11 @@
 		"description": "Adds a link to create issues from anywhere in a repository.",
 		"screenshot": "https://github-production-user-asset-6210df.s3.amazonaws.com/1402241/274816033-820ec518-049d-4248-9f8a-27b9423e350b.png"
 	},
+	{
+		"id": "quick-reactions",
+		"description": "Shows a reaction optimistically",
+		"screenshot": "https://user-images.githubusercontent.com/1402241/236628453-8b646178-b838-44a3-9541-0a9b5f54a84a.png"
```

Comment:
I'm just commenting for an issue I'm opening.
Will edit comment with issue number.